sync/class.jetpack-sync-defaults.php

```diff
@@ -81,7 +81,15 @@ class Jetpack_Sync_Defaults {
 		'disabled_reblogs',
 		'jetpack_comment_likes_enabled',
 		'twitter_via',
-		'twitter-cards-site-tag'
+		'twitter-cards-site-tag',
+		'wpcom_publish_posts_with_markdown',
```

Comment:
I have added test where we test setting the default options. 
Should I add a test  that would show us if we haven't tested an option? 

So that a test would have to add the option to test for the default options for the test to pass?